app/assets/stylesheets/application.css

```diff
@@ -293,12 +288,139 @@ canvas{
   left: 400px;
   left:30%;
   margin: 0px auto;
+  background: transparent;
+  margin: 0px auto;
+  left:30%;
 }
 
 #myCanvas{
   z-index: 2;
+  border-style: solid;
 }
 
 #canvasLava{
   z-index: 0;
 }
+.skrollr-desktop body {
+    height:100% !important;
+}
+
+
+/*  Styles for this example */
+.container-rules{
+  width:100%;
+
+}
+
+.features-list{
+    padding: 50px 0;
+    background-color:#201D1C;
+    color: #FCE9C7;
+    font-size: 18px;
+    min-height: 60vh;
+    text-align: center;
+}
+
+.features-list div{
+  background-color: black;
+  opacity: 50%;
+  display: inline-block;
+  width: 20%;
+  margin: 5px;
+  border: 2px solid #AB2A1A;
+  padding: 5px;
+    /*box-shadow: 1px 1px 1px #ccc, 0 0 30px #eee inset;*/
+}
+
+.features-list div i{
+    font-size: 50px;
+    margin-bottom: 10px;
+    color: #EB3A23;
+
+}
+.footer{
+  position: absolute;
+  bottom: 0;
+  background-color: #35547D;
+  color:black;
+  width: 100%;
+}
+.about-us{
+
+}
+
+.about-us h2{
+    font-size: 42px;
+    margin-bottom: 80px;
+    font-weight: normal;
+}
+
+.about-us div{
+    text-align: center;
+    width: 25%;
+    margin: 40px 30px;
+    display: inline-block;
+}
+
+
+.about-us div img{
+    border-radius: 50%;
+    width: 30%;
+}
+
+.about-us a.attribution{
+    display: block;
+    font-size: 13px;
+    text-decoration: none;
+    color: inherit;
+    width: 280px;
+    margin: 50px auto 10px;
+    opacity: 0.5;
+}
+
+
+@media (max-width: 800px){
+
+    .about-us div{
+        width: 45%;
+        margin: 20px 0;
+    }
+
+}
+.well {height:200px;}
+
+/* demo */
+
+@media (min-width:768px) {
+    .inline-block-row {
+        word-spacing: -1em;
+        letter-spacing: -1em;
+        overflow:hidden;
+    }
+    .inline-block-row .col-sm-4 {
+        word-spacing: normal;
+        vertical-align: top;
+        letter-spacing: normal;
+        display: inline-block;
+        float:none;
+    }
+}
```

Comment:
I'm not 100% sure, but I heard that bootstrap already has some classes that will take care of different media sizing for us. This looks good though, just a suggestion, no need to change this if it's working.